pkg/node/node.go

```diff
@@ -2274,3 +2279,19 @@ func ExtractPort(urlString string) (string, error) {
 
 	return port, nil
 }
+
+func removeVolumePrefixFromName(volumeName string) string {
```

Comment:
I think it would be more reliable to find the last occurrence of `csi-volume-prefix` and trim everything before that.
For example, if 
auth_prefix  = `csi`
csi_vol_preofix = `csi`
the volume name might end up being `csi-csi-abc568ths`. In this case, the current logic would return the full name as-is, which might not be ideal.